auslib/dockerflow.py

```diff
@@ -29,7 +32,7 @@ def heartbeat():
         depend on services like the database to also ensure they are healthy."""
         # Counting the rules should be a trivial enough operation that it won't
         # cause notable load, but will verify that the database works.
-        dbo.rules.countRules()
+        dbo.dockerflow.incrementWatchdogValue(changed_by=DOCKERFLOW_DB_USER)
```

Comment:
One thing I just realized here is that we can't use incrementWatchdogValue on the public app (because it correctly doesn't have write permissions). We should continue using countRules() there.